modules/ivc_champva/app/form_mappings/vha_10_10d.json.erb

```diff
@@ -14,51 +14,51 @@
-  "form1[0].#subform[0].EmailAddress2[0]":               "<%= form.data['applicants'][1]&.dig('email') %>",
-  "form1[0].#subform[0].PhoneNumber2[0]":                "<%= form.data['applicants'][1]&.dig('phone_number') %>",
-  "form1[0].#subform[0].Gender2[0]":                     "<%= form.data['applicants'][1]&.dig('gender') == 'male' ? 0 : 1 %>",
-  "form1[0].#subform[0].EnrolledMedicare[1]":            "<%= form.data['applicants'][1]&.dig('is_enrolled_in_medicare') ? 1 : 0  %>",
-  "form1[0].#subform[0].HasOtherInsurance[1]":           "<%= form.data['applicants'][1]&.dig('has_other_health_insurance') ? 1 : 0  %>",
-  "form1[0].#subform[0].RelationshipToVeteran2[0]":      "<%= form.data['applicants'][1]&.dig('vet_relationship') %>",
-  "form1[0].#subform[0].State2[0]":                      "<%= form.data['applicants'][1]&.dig('address', 'state') %>",
-  "form1[0].#subform[0].LastName3[0]":                   "<%= form.data['applicants'][2]&.dig('full_name', 'last') %>",
-  "form1[0].#subform[0].FirstName3[0]":                  "<%= form.data['applicants'][2]&.dig('full_name', 'first') %>",
-  "form1[0].#subform[0].MI3[0]":                         "<%= form.data['applicants'][2]&.dig('full_name', 'middle')&.first %>",
-  "form1[0].#subform[0].SSN3[0]":                        "<%= form.data['applicants'][2]&.dig('ssn_or_tin') %>",
-  "form1[0].#subform[0].StreetAddress3[0]":              "<%= form.data['applicants'][2]&.dig('address', 'street') %>",
-  "form1[0].#subform[0].City3[0]":                       "<%= form.data['applicants'][2]&.dig('address', 'city') %>",
-  "form1[0].#subform[0].ZipCode3[0]":                    "<%= form.data['applicants'][2]&.dig('address', 'postal_code') %>",
-  "form1[0].#subform[0].DateOfBirth3[0]":                "<%= form.data['applicants'][2]&.dig('date_of_birth')  %>",
-  "form1[0].#subform[0].EmailAddress3[0]":               "<%= form.data['applicants'][2]&.dig('email') %>",
-  "form1[0].#subform[0].PhoneNumber3[0]":                "<%= form.data['applicants'][2]&.dig('phone_number') %>",
-  "form1[0].#subform[0].Gender3[0]":                     "<%= form.data['applicants'][2]&.dig('gender') == 'male' ? 0 : 1 %>",
-  "form1[0].#subform[0].EnrolledMedicare[2]":            "<%= form.data['applicants'][2]&.dig('is_enrolled_in_medicare') ? 1 : 0  %>",
-  "form1[0].#subform[0].HasOtherInsurance[2]":           "<%= form.data['applicants'][2]&.dig('has_other_health_insurance') ? 1 : 0  %>",
-  "form1[0].#subform[0].RelationshipToVeteran3[0]":      "<%= form.data['applicants'][2]&.dig('vet_relationship') %>",
-  "form1[0].#subform[0].State3[0]":                      "<%= form.data['applicants'][2]&.dig('address', 'state') %>",
+  "form1[0].#subform[0].LastName1[0]":                   "<%= form.data.dig('applicants', 0, 'applicant_name', 'last') %>",
+  "form1[0].#subform[0].FirstName1[0]":                  "<%= form.data.dig('applicants', 0, 'applicant_name', 'first') %>",
```

Comment:
Changed the property names we're accessing for all applicant properties because the submit transformer ([updated here](https://github.com/department-of-veterans-affairs/vets-website/pull/31419)) is no longer changing property names unnecessarily.